LongWarOfTheChosen/Localization/XComGame.int

```diff
@@ -988,42 +988,61 @@ TacticalText="<Bullet/> Coil weapons automatically pierce (ignore) 1 enemy armor
+; Bullpup changes and new templates
+[Bullpup_CV X2WeaponTemplate]
+TacticalText="<Bullet/> Bullpup-style are extremely effective at close range and are designed for hit-and-run tactics.\n<Bullet/> They also increase the Skirmisher's mobility and improve their ability to slip by ADVENT forces undetected.\n<Bullet/> Use upgrades looted from enemies to improve the Bullpup's tactical effectiveness."
+
 [Bullpup_LS X2WeaponTemplate]
 FriendlyName="Kal-13 \"Bullpup\""
 FriendlyNamePlural="Kal-13 Bullpups"
-BriefSummary="We can manufacture a modified Laser SMG That can be used by Skirmishers. While Dr.Tygan Has correctly stated that this is in fact not a bullpup, it's still a small, mobile gun that shoots lasers and that works for me."
-TacticalText="<Bullet/> Bullpup-style weapons are extremely effective at close range, but their accuracy drops off quickly at longer range.\n<Bullet/> Use upgrades looted from enemies to improve the Kal-15 Bullpup's tactical effectiveness."
+BriefSummary="We can manufacture a modified Laser SMG that can be used by Skirmishers. While Dr.Tygan has correctly stated that this is in fact not a bullpup, its compact frame and the additional tweaks we've made are enough for them to seriously consider its use in battle."
+TacticalText="<Bullet/> Laser weapons receive bonus aim.\n<Bullet/> Bullpup-style weapons are extremely effective at close range and are designed for hit-and-run tactics.\n<Bullet/> They also increase the Skirmisher's mobility and improve their ability to slip by ADVENT forces undetected.\n<Bullet/> Use upgrades looted from enemies to improve the Kal-13 Bullpup's tactical effectiveness."
 AbilityDescName="bullpup"
 
+[Bullpup_MG X2WeaponTemplate]
+TacticalText="<Bullet/> Bullpup-style weapons are extremely effective at close range and are designed for hit-and-run tactics.\n<Bullet/> They also increase the Skirmisher's mobility and improve their ability to slip by ADVENT forces undetected.\n<Bullet/> Use upgrades looted from enemies to improve the Kal-15 Bullpup's tactical effectiveness."
+
 [Bullpup_CG X2WeaponTemplate]
 FriendlyName="Kal-69 \"Bullpup\""
 FriendlyNamePlural="Kal-69 Bullpups"
-BriefSummary="We can manufacture a modified Laser SMG That can be used by Skirmishers. While Dr.Tygan Has correctly stated that this is in fact not a bullpup, it's still a small, mobile gun that fires rounds sheathed in ablative plastic and that works for me."
-TacticalText="<Bullet/> Bullpup-style weapons are extremely effective at close range, but their accuracy drops off quickly at longer range.\n<Bullet/> Use upgrades looted from enemies to improve the Kal-69 Bullpup's tactical effectiveness."
+BriefSummary="As the bullpup frame the skirmishers employ for their weapons has proven too weak to be able control the recoil of a coil weapon and withstand potential damage, we've instead decided to adapt a model of our Subcoilers for field use by skirmishers, just like we did with the Kal-13 \"Bullpup\"."
```

Comment:
Inconsistency of capitalisation on Bullpup. Also, why the quotes around Bullpup in this case?

Reply:
Like the laser bullpup, this weapon isnt actually a bullpup, so calling it one is only a nickname. Im adding the quotes to the plural though so its consistent.

Reply:
Fair enough, makes sense. I think it should probably still be "Bullpup frame".